css/my_website.css

```diff
@@ -0,0 +1,52 @@
+body{
+    margin:0px;
+    background-color:rgb(231, 231, 231);
+    color:rgb(65, 65, 65);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size:20px;
+    line-height:normal;
+}
+.main-header{
+    background-color:#003F87;
+    color:#ffffff;
+    height: 75px;
+    padding: 0em;
+    width:100%;
+    overflow: hidden;
+}
+.text{
+    color: black;
+    padding:1em;
+    margin-left:1em;
+    margin-right:1em;
+    outline-color: rgb(255, 0, 0);
+    background-color:white;
+    outline-style: solid;
+    outline-width: 1em;
+}
+#title{
+    margin:5px;
+    padding:10px;
```

Comment:
Think about when and why you are using margin vs padding.
[See this answer on stack overflow.](https://stackoverflow.com/questions/2189452/when-to-use-margin-vs-padding-in-css)

Reply:
Ok I made it a little simpler